docs/_docs/development/dokku.md

```diff
@@ -0,0 +1,43 @@
+
+
+# Dokku setup
+SSH into your new Dokku server.
+
+# Install redis on dokku
+The FreeGenes app uses redis for caching. Dokku has a convenient 
+
+dokku plugin:install https://github.com/dokku/dokku-redis.git redis
```

Comment:
All these sections need code blocks around them, e.g.,

```bash
dokku ...
```
Otherwise they won't render properly in the site. Did you mean to add a link to this deployment option, or not yet?

Reply:
To be honest, I have no clue how that file is even there. Should have read commit message more carefully, just saw the deletion and thought "hey that's good"

Reply:
okay now we're good! When tests pass, we'll merge this in.